recipes/thorvg/all/test_package/test_package.cpp

```diff
@@ -4,7 +4,7 @@ int main() {
     const int WIDTH = 800;
     const int HEIGHT = 600;
 
-    tvg::Initializer::init(tvg::CanvasEngine::Sw, 0);
+    tvg::Initializer::init(0, tvg::CanvasEngine::Sw);
```

Comment:
We need some context here 😅

Reply:
@jcar87 `tvg::Initializer::init` interface changed in this version 
https://github.com/thorvg/thorvg/blob/eb1208414d2da5d48836ac62de7cbaf1d3e30d08/src/renderer/tvgInitializer.cpp#L89

Reply:
This change would need to be conditional on the version, else the old versions would fail with the new changes :)